packages/ui/src/components/FormRenderer/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,136 @@
+/** *******************************************************************************************************************
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+      http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.                                                                              *
+ ******************************************************************************************************************** */
+import { FC, memo } from 'react';
+import CheckboxComponent from '@cloudscape-design/components/checkbox';
+import FormField from '@cloudscape-design/components/form-field';
+import SpaceBetween from '@cloudscape-design/components/space-between';
+import useFieldApi, { UseFieldApiConfig } from '@data-driven-forms/react-form-renderer/use-field-api';
+import { FieldInputProps } from 'react-final-form';
+import useUniqueId from '../../../../hooks/useUniqueId';
+import getErrorText from '../../utils/getErrorText';
+import { Option } from '../../types';
+
+interface CheckboxMappingProps {
+    input: FieldInputProps<any, HTMLInputElement>;
+    option: Option;
+    value?: Option['value'][];
+    isDisabled?: boolean;
+}
+
+const CheckboxMapping: FC<CheckboxMappingProps> = ({ input, option, isDisabled, ...rest }) => {
+    return (
+        <CheckboxComponent
+            disabled={isDisabled}
+            {...input}
+            {...option}
+            checked={(input.value || []).includes(option.value)}
+            controlId={`${input.name}-${option.value}`}
+            onBlur={() => input.onBlur()}
+            onFocus={() => input.onFocus()}
```

Comment:
```suggestion
            onBlur={input.onBlur}
            onFocus={input.onFocus}
```

Reply:
The type for both event handlers are having different signatures. This is to avoid type casting.